### PR TITLE
Improve error message collect for Zendesk plugin

### PR DESCRIPTION
**Pull request by pauloouriques**
- add better messages to Zendesk plugin retrieving the `result` field from the API client response.

### CODE REVIEW

File: plugins/@grouparoo/zendesk/src/lib/export/exportRecord.ts [hunk 1]
```diff
@@ -148,6 +148,8 @@ export const exportRecord: ExportRecordPluginMethod = async ({
 
     return { success: true };
   } catch (error) {
+    const errorMessage = error?.result?.toString();
```
Comment (bleonard) [inline plugins/@grouparoo/zendesk/src/lib/export/exportRecord.ts:151]: what's an example of this? I'm curious if it's losing information or not. (like it used to say `422`)

**Review by bleonard — CHANGES_REQUESTED** [review]
I'd expect to see some test changes on a failed one that is checking for a better message.

File: plugins/@grouparoo/zendesk/src/lib/export/exportRecord.ts (hunk 1 above)
Comment (pauloouriques) [reply plugins/@grouparoo/zendesk/src/lib/export/exportRecord.ts:151]: The `errorMessage` for a 429 is: _You have exceeded the rate limit for updating a single unique user. user email: othergrouparoo@demo.com id:  external_id: grouparoo123_

Comment (bleonard) [reply plugins/@grouparoo/zendesk/src/lib/export/exportRecord.ts:151]: That's cool. Let's verify one for a `422` as well. I think you'd sync someone with a bad or duplicate email

**Review by bleonard — APPROVED** [review]
Thanks!

**Pull request merged**